### PR TITLE
Add ASIO rights statement

### PR DESCRIPTION
**Pull request by gilgongo**
**Short description of changes**

1. Add logo and rights text to footer
2. Add statement about binary distribution on Windows download page.

Also removed nested footer from "wiki" and moved CC rights statement to docs nav to remove any confusion about what was CC. 

**Context: Fixes an issue? Related issues**

See https://github.com/jamulussoftware/jamulus/issues/2417 

**Status of this Pull Request**

I'm sure there are 101 ways of doing all this better with CSS. Feel free to suggest how.

**What is missing until this pull request can be merged?**

Other than 101 suggestions about CSS, the merge needs to coincide with the things doing to the installer an about screens as per https://github.com/jamulussoftware/jamulus/issues/2417 

**Does this need translation?**

Not sure? Perhaps.

## Checklist

- [ ] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: wiki/en/Installation-for-Windows.md [hunk 1]
```diff
@@ -20,11 +20,12 @@ Upgrading? You may want to [back up your configuration](Software-Manual#backing-
 1. **Run Jamulus**. Now you should be able to use Jamulus just like any other application.
 1. **Set up your sound card**. When you're done, you need to set up your audio hardware. And if you are using ASIO4ALL, have a look at how to set that up below.
 
+_You are not permitted to redistribute this binary without acquiring a [licence agreement from Steinberg](https://www.steinberg.net/developers/)._
```
Comment (ann0see) [inline wiki/en/Installation-for-Windows.md]:
```suggestion
_Please note that you are not permitted to redistribute this binary without acquiring a [licence agreement from Steinberg](https://www.steinberg.net/developers/)._
```

Maybe formulate it positively

```suggestion
_Note: You need a [licence agreement from Steinberg](https://www.steinberg.net/developers/) if you want to distribute this binary._
```

File: wiki/en/Installation-for-Windows.md [hunk 2]
```diff
@@ -34,8 +35,6 @@ You can try two versions of ASIO4ALL. ASIO4ALL v2.14 includes a workaround for a
 
 [ASIO4ALL v2.14 Download](https://github.com/jamulussoftware/assets/raw/main/ASIO4ALL/v2.14/ASIO4ALL_2_14_English.exe){: .button target="_blank" rel="noopener noreferrer"}
 
-[ASIO4ALL website](https://www.asio4all.org/){: target="_blank" rel="noopener noreferrer"}
-
```
Comment (ann0see) [inline wiki/en/Installation-for-Windows.md]: Why did you remove this?

File: _layouts/wiki.html [hunk 3]
```diff
@@ -15,14 +15,6 @@
...
       <noscript class="red"><strong>{{ site.data.general.tNoJSEnabled }}</strong></noscript>
       <main lang="{{ page.lang }}">{{ content }}</main>
-      <div id="ftext" class="fx-txt-center">
-        {% capture footertext %}
-        {% if site.active_lang %}
-        {% include wiki/{{ site.active_lang }}/footertext.md %}
-        {% endif %}
-        {% endcapture %}
-        {{ footertext | markdownify }}
-      </div>
```
Comment (ann0see) [inline _layouts/wiki.html:25]: If this is removed, the respective CSS should also be removed. But I don't quite understand why we need to remove it.

File: wiki/en/misc/footertext.md [hunk 4]
```diff
@@ -1,6 +0,0 @@
-
```
Comment (ann0see) [inline wiki/en/misc/footertext.md:1]: Why is this removed?

File: wiki/en/misc/general.yml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 footer:
-  copyright:         Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/jamulussoftware/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a>
+  copyright:         Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/jamulussoftware/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a> - want to <a href="/wiki/Contribution">get involved?</a> <div id="asio" style="margin-top:20px;">ASIO is a trademark and software of Steinberg Media Technologies GmbH <img alt="ASIO logo" src="/assets/img/asio.jpg" style="display:block;margin-left:auto;margin-right:auto;width:70px;height:auto;" loading="lazy"></div>
```
Comment (ann0see) [inline wiki/en/misc/general.yml]: I'd add it to the footer in the footer inclusion file: https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/_includes/footer.html

**Review by ann0see — COMMENTED** [review]
I think a designer should review this.

File: wiki/en/Installation-for-Windows.md (hunk 2 above)
Comment (gilgongo) [reply wiki/en/Installation-for-Windows.md]: We're linking to them elsewhere, and I don't want to attract their attention any more than necessary. But I also don't consider a PR complete unless I've deleted something :-)

File: _layouts/wiki.html (hunk 3 above)
Comment (gilgongo) [reply _layouts/wiki.html:25]: I don't know enough about the CSS to remove that CSS. The reason I removed the "wiki footer" was so that we just have one footer on the website, don't confuse CC with ASIO and Jamulus rights, and to close the survey which needs removing now until I get round to doing a better one.

File: wiki/en/misc/footertext.md (hunk 4 above)
Comment (gilgongo) [reply wiki/en/misc/footertext.md:1]: If that's the same thing as the "wiki" footer, then see above (I get confused with all the different files controlling different things).

File: wiki/en/misc/general.yml (hunk 5 above)
Comment (gilgongo) [reply wiki/en/misc/general.yml]: Sorry, what do you mean? Do you mean footer.html is a better place for it than general.yml?

Comment (ann0see) [reply wiki/en/misc/general.yml]: Yes

File: wiki/en/misc/general.yml [hunk 6]
```diff
@@ -9,12 +9,12 @@ nav:
...
   altJamulusIcon:    "Jamulus Icon. Links to homepage"
 wiki:
-  titleAdd:          "– Jamulus Wiki"
+  titleAdd:          "– Jamulus Website"
 kb:
   titleAdd:          "– Jamulus Knowledge Base"
   by:                "by"
   kbintro:           "<strong>Community knowledge base</strong><br /> Feel free to add your own article!"
-  mainpage:          "KB overview"
+  mainpage:          "Knolwege Base overview"
```
Comment (pljones) [inline wiki/en/misc/general.yml]: > Knolwege

"Knowledge".

Comment (gilgongo) [reply wiki/en/misc/general.yml]: I fixed that earlier. How come it isn't appearing? https://github.com/gilgongo/jamuluswebsite/blob/d61a7ca28aedf85ee6c59ce0356118e666829b89/wiki/en/misc/general.yml

Comment (hoffie) [reply wiki/en/misc/general.yml]: It does appear here now:
https://github.com/jamulussoftware/jamuluswebsite/pull/721/files

I've only two explanations for this:
- @pljones might have started his review before your commit, but might have submitted past your commit.
- A Github cache coherency bug. It's still strange that this thread does not carry an "Outdated" tag. I think (but I'm no longer sure!) that I saw the typo fix missing when visiting the above URL.

During testing, I visited the following URL. It *might* have cleared some Github cache:
https://github.com/jamulussoftware/jamuluswebsite/compare/next-release..gilgongo:asio-2

File: _includes/footer.html [hunk 7]
```diff
@@ -18,7 +18,10 @@
     </a>
   </div>
   <div id="copyright" class="fx-col-100-xs fx-row-center-xs">
-    {{ site.data.general.footer.copyright }}
+	Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/jamulussoftware/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a> - want to <a href="/wiki/Contribution">get involved?</a> 
+	<div id="asio" style="margin-top:20px;">
```
Comment (pljones) [inline _includes/footer.html:22]: Should this really be nested inside the `copyright` div or a separate `asio` div (as it's not an assertion but an acknowledgement - two distinct entities)?

File: assets/css/wiki.css [hunk 8]
```diff
@@ -82,3 +66,19 @@ details > summary {
   color: #1882A1;
   cursor: pointer;
 }
+
```
Comment (pljones) [inline assets/css/wiki.css:69]: If the styling for `cc` and `asio` divs is intentionally identical, can it not be declared once in a more general stylesheet, or is there only one per page?

File: wiki/en/Installation-for-Windows.md [hunk 9]
```diff
@@ -20,11 +20,12 @@ Upgrading? You may want to [back up your configuration](Software-Manual#backing-
 1. **Run Jamulus**. Now you should be able to use Jamulus just like any other application.
 1. **Set up your sound card**. When you're done, you need to set up your audio hardware. And if you are using ASIO4ALL, have a look at how to set that up below.
 
+_Please note that you are not permitted to redistribute this binary without acquiring a [licence agreement from Steinberg](https://www.steinberg.net/developers/)._
+
 ***
 
-## ASIO
+## Jamulus uses [ASIO](https://en.wikipedia.org/wiki/Audio_Stream_Input/Output)
```
Comment (pljones) [inline wiki/en/Installation-for-Windows.md:27]: Have you read the Wikipedia page?  It's dire...  I'd be tempted to link Steinberg directly but it's likely to move.

File: wiki/en/Installation-for-Windows.md (hunk 2 above)
Comment (pljones) [reply wiki/en/Installation-for-Windows.md]: And `wiki/en/misc/footertext.md` wasn't enough?

File: wiki/en/Installation-for-Windows.md (hunk 9 above)
Comment (gilgongo) [reply wiki/en/Installation-for-Windows.md:27]: Like I say, I'm slightly reluctant to link to Steinberg, but maybe I'm just paranoid.

File: assets/css/wiki.css (hunk 8 above)
Comment (gilgongo) [reply assets/css/wiki.css:69]: Think there has to be one per page (at least, there's wiki.css, home.css and kb.css)

File: _includes/footer.html (hunk 7 above)
Comment (gilgongo) [reply _includes/footer.html:22]: Dunno. But I was expecting suggestions, certainly.

File: wiki/en/Installation-for-Windows.md (hunk 2 above)
Comment (gilgongo) [reply wiki/en/Installation-for-Windows.md]: No, I had reasons to delete that :-)

Ha - I've just realised this is a link to ASIO4all not Stienberg. Sorry. Yes, we can link to them and all their ad trackers too if we really want :-)

File: wiki/en/Installation-for-Windows.md (hunk 9 above)
Comment (pljones) [reply wiki/en/Installation-for-Windows.md:27]: I'm only reluctant because it'll mean we get link breakage (do we run a link check as part of the doc build?).

Comment (ann0see) [reply wiki/en/Installation-for-Windows.md:27]: > do we run a link check as part of the doc build?

No. I gave up on that a while ago.

Comment (gilgongo) [reply wiki/en/Installation-for-Windows.md:27]: I've put the link back in to ASIO4all now anyway.

Comment (gilgongo) [reply wiki/en/Installation-for-Windows.md:27]: > I'm only reluctant because it'll mean we get link breakage (do we run a link check as part of the doc build?).

We do manually.

**Comment by gilgongo** [comment]
Any objections to merge? It does the job (for the website), and if we want to gold-plate the CSS we can do so later.

File: assets/css/wiki.css (hunk 8 above)
Comment (ann0see) [reply assets/css/wiki.css:69]: There’s also a general css file which gets included in every page. I think it’s fw.css.

File: wiki/en/misc/general.yml [hunk 10]
```diff
@@ -1,5 +1,5 @@
 footer:
-  copyright:         Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/jamulussoftware/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a>
+  copyright:
```
Comment (ann0see) [inline wiki/en/misc/general.yml:2]: Remove this and all occurrences? But this implies we can’t translate the copyright

Comment (gilgongo) [reply wiki/en/misc/general.yml:2]: I don't think we're allowed to translate it, are we? At least, the language given in the agreement is English. Interesting point though since the jurisdiction is Germany :-) But I assume if they had other legally binding translations, they'd have listed those.

Comment (gilgongo) [reply wiki/en/misc/general.yml:2]: Oh sorry, this is ours, not ASIO's. Hm.
